x-pack/plugins/alerts/server/alerts_client.test.ts

```diff
@@ -689,9 +804,9 @@ describe('create()', () => {
     const data = getMockData();
     alertsClientParams.createAPIKey.mockResolvedValueOnce({
       apiKeysEnabled: true,
-      result: { id: '123', api_key: 'abc' },
+      result: { id: '123', name: '123', api_key: 'abc' },
```

Comment:
I don't know - I added `name` because the Type requires it (and the tests didn't have proper typing on them before.
This isn't an addition made in this PR but just a correction of something that was missed in a previous PR.
I'm not actually sure where the `name` field comes form 🤔